lib/Conch/Route/Datacenter.pm

```diff
@@ -127,30 +349,198 @@ sub routes {
+
+=over 4
+
+=item * Requires Admin Authentication
+
+=item * Request Schema: L<RackCreate|https://conch.joyent.us/schema/request/RackCreate>
+
+=back
+
+=cut
+
         # POST /rack
         $rack->post('/')->to('#create');
 
         my $with_rack = $rack->under('/<rack_id:uuid>')->to('#find_rack');
 
+
+=head3 C<GET /rack/:rack_id>
+
+Get a single rack.
+
+=over 4
+
+=item * Requires Authentication
+
+=item * Response Schema: L<Rack|https://conch.joyent.us/schema/response/Rack>
+
+=back
+
+=cut
+
         # GET /rack/:rack_id
         $with_rack->get('/')->to('#get');
+
+=head3 C<POST /rack/:rack_id>
+
+Update an existing rack. Returns a C<400 BAD REQUEST> if the new room or rack
+role does not exist. Returns a C<400 BAD REQUEST> if the new rack size is
+smaller than the currently assigned number of rack units. Upon success returns
+a C<303 SEE OTHER> with the Location of the updated rack.
+
+=over 4
+
+=item * Requires Authentication
+
+=item * Request Schema: L<RackUpdate|https://conch.joyent.us/schema/request/RackUpdate>
+
+=back
+
+=cut
+
         # POST /rack/:rack_id
         $with_rack->post('/')->to('#update');
+
+=head3 C<DELETE /rack/:rack_id>
+
+Delete a rack. Returns a C<400 BAD REQUEST> if the rack is still referenced by
+a rack layout. Returns a C<204 NO CONTENT> upon success.
+
+=over 4
+
+=item * Requires Authentication
+
+=back
+
+=cut
+
         # DELETE /rack/:rack_id
         $with_rack->delete('/')->to('#delete');
+
+=head3 C<GET /rack/:rack_id/layouts>
+
+Gets all the layouts for the specified rack.
+
+=over 4
+
+=item * Requires Authentication
+
+=item * Response Schema: L<RackLayouts|https://conch.joyent.us/schema/response/RackLayouts>
+
+=back
+
+=cut
+
         # GET /rack/:rack_id/layouts
         $with_rack->get('/layouts')->to('#layouts');
 
+
+=head3 C<GET /rack/:rack_id/assignment>
+
+Gets all the rack layout assignments (including occupying devices) for the
+specified rack.
+
+=over 4
+
+=item * Requires Authentication
+
+=item * Response Schema: L<RackAssignments|https://conch.joyent.us/schema/response/RackAssignments>
+
+=back
+
+=cut
+
         # GET /rack/:rack_id/assignment
         $with_rack->get('/assignment')->to('#get_assignment');
+
+=head3 C<POST /rack/:rack_id/assignment>
+
+Assigns devices to rack layouts, also optionally updating asset tags. Returns a
+C<400 BAD REQUEST> if there are missing layout assignments. Returns a C<400 BAD
+REQUEST> if we try to assign a device to an already filled location, or if the
+device already has an assigned location. Returns a C<303 SEE OTHER> with the
+location of the updated rack assignment.
+
+=over 4
+
+=item * Requires Authentication
+
+=item * Request Schema: L<RackAssignmentUpdates|https://conch.joyent.us/schema/request/RackAssignmentUpdates>
+
+=back
+
+=cut
+
         # POST /rack/:rack_id/assignment
         $with_rack->post('/assignment')->to('#set_assignment');
+
+=head3 C<DELETE /rack/:rack_id/assignment>
+
+B<THIS METHOD REQUIRES A REQUEST BODY>
```

Comment:
I don't think this needs to be bold and all caps. A simple statement will suffice.

Reply:
Reduced this to just a statement. I left it bold and all caps because a request body on a DELETE is very unusual.